vac.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Main
+---
+# Vac researches peer-to-peer, private, censorship resistant communication.
+
+
+## About Vac
+Vac is an R&D org creating modular p2p protocols for private, secure, censorship resistant communication.
+Researching All Layers
+
+### Researching All Layers
+Concerned with all layers in the stack including underlying transports, p2p overlays and routing, initial trust establishment, and semantics for things like group chat.
+
+### Peer-to-Peer
+The protocols we work on are pure peer-to-peer, and aim to minimize centralization. This too is in opposition to many initiatives in the secure messaging space.
+
+### Security, Privacy, Censorship Resistance
+Assumes the basics such as end-to-end encryption, forward secrecy, avoiding MITM-attacks. Vac also places a premium on privacy and censorship resistance from port blocking, traffic analysis, and similar.
+
+### Modular System
+Provide options at each layer in the stack, instead of having a tightly coupled set of protocols. Allowing developers to choose what they use and ensure they know each choice comes with different trade-offs.
+
+
+### Generalized Messaging
+This includes both human to human communication, as well as machine to machine communication. Texting, data transfer, financial transactions, state channels and more.
+
+## Principles
+[missing content]
```

Comment:
This can be copy paste from https://our.status.im/our-principles/ starting from

Reply:
@oskarth I have made these changes and will open for review soon.